# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/squashfs-tools/hotfix-no-pager.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

PAGER cannot have spaces or tabs!
If you want to do this, please use a wrapper script!

Just no.

--- squashfs-tools-4.7/squashfs-tools/print_pager.c.vanilla	2025-06-14 19:39:28.078602295 +0200
+++ squashfs-tools-4.7/squashfs-tools/print_pager.c	2025-06-14 19:43:44.423198681 +0200
@@ -239,6 +239,8 @@
 
 FILE *exec_pager(pid_t *process)
 {
+	return stdout;
+
 	FILE *file;
 	int res, pipefd[2];
 	pid_t child;
--- squashfs-tools-4.7/squashfs-tools/print_pager.c.vanilla	2025-06-14 19:44:59.162874692 +0200
+++ squashfs-tools-4.7/squashfs-tools/print_pager.c	2025-06-14 19:50:24.423116037 +0200
@@ -77,6 +77,7 @@
 
 int check_and_set_pager(char *pager)
 {
+	return TRUE;
 	int i, length = strlen(pager);
 	char *base;
 
